### DISCUSSION TITLE
bollo a carico del cliente e rivalsa inps

### DISCUSSION THREAD

**Discussion by zpj61** [fattureincloud/api#303] [Q&A] [answered]
Buongiorno, ho un problema applicando il bollo a carico del cliente come descritto in altro post inserendo l'importo in items_list.
A questo punto se richiedo in [Get New Issued Document Totals] ottengo:
imponibile = 102
rivalsa INPS = 4,08 che implica sia stata calcolata su 102
percentuale tassabile = 106,08
ritenuta acconto = 21,22 
ma il bollo è considerato parte del ricavo solo in regime forfettario mentre in questo caso è un regime ordinario. 
Cosa non quadra?
Grazie

**Comment by emajo** [comment, marked as answer]
Ciao @zpj61, effettivamente non abbiamo inserito nella guida che è necessario settare il flag `"not_taxable": true`.

Grazie,
Emanuele.
